components/kyma-environment-broker/internal/process/update/initialisation_step.go

```diff
@@ -64,11 +64,20 @@ func (s *InitialisationStep) Run(operation internal.UpdatingOperation, log logru
 		if err != nil {
 			return operation, time.Second, nil
 		}
+		if operation.ProvisioningParameters.ErsContext.SMOperatorCredentials != nil {
+			instance.Parameters.ErsContext.SMOperatorCredentials = operation.ProvisioningParameters.ErsContext.SMOperatorCredentials
```

Comment:
both update and upgrade queues need to take the SMOperatorCreds from previous operations if the current operation doesn't have them.